platform/podjs/README.md

```diff
@@ -20,21 +20,48 @@ feature. While we could support the creation of a container and a
 mechanism for loading features in `pod.js` without too much trouble,
 it is currently not implemented.
 
-The `polyOut` and `polyNav` interfaces are not implemented at the
-moment. The former because we don't use it yet, the latter because it
-is currently not part of the `Pod` interface, which we aim to rectify
-soon.
-
 ## Building
 
-Simply run `build.js` in the repository root.
+Simply run `./build.js` in the repository root.
+
+## How to set up a feature to build with podjs
+
+Follow these instructions to do so:
+
+1.  You need to point to `podjs` from the `polypoly-eu` repository and
+    add it as a "development dependency" on your project by running:
+
+    `$npm i <path-that-points-to-platform/podjs --D`
+
+    So, you end up having on your `package.json` the following:
+
+    ```json
+       "devDependencies": {
+          "@polypoly-eu/podjs": "file: <path-that-points-to-platform/podjs>",
+          ...
+       },
+
+    ```
+
+2.  The main step is to copy `pod.js` in your project. For this, please, check the `example` feature as guideline. You should be able to do it with `rollup` tooling (same way as in `example`) or any other script mechanism you desire.
```

Comment:
I think that @fhd suggested `demo` instead. Any one of them is good enough, I guess.

Reply:
No, I actually changed my mind on that later - turns out _demo_ doesn't use Rollup! :D